NetworkPkg/Application/VConfig/VConfig.c

```diff
@@ -654,18 +654,40 @@ VlanConfigMain (
 
   if (ShellCommandLineGetFlag (List, L"-l")) {
     Str = ShellCommandLineGetValue (List, L"-l");
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (Str == NULL) {
+      ShellPrintHiiEx (-1, -1, NULL, STRING_TOKEN (STR_VCONFIG_NO_IF), mHiiHandle);
+      goto Exit;
+    }
+
+    // MU_CHANGE [END] - CodeQL change
     DisplayVlan ((CHAR16 *)Str);
     goto Exit;
   }
 
   if (ShellCommandLineGetFlag (List, L"-a")) {
     Str = ShellCommandLineGetValue (List, L"-a");
-    AddVlan ((CHAR16 *)Str);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (Str == NULL) {
+      ShellPrintHiiEx (-1, -1, NULL, STRING_TOKEN (STR_VCONFIG_NO_IF), mHiiHandle);
+      goto Exit;
+    }
+
+    // MU_CHANGE [END] - CodeQL change
+    DisplayVlan (NULL);

```

Comment:
Should this be AddVlan (deleted line 663 above) not DisplayVlan?

Reply:
Updated in PR commit 9108f8b6b0.